### PR TITLE
Pa02

### PR DESCRIPTION
**Pull request by MarkusOla**

### CODE REVIEW

**Comment by huynhngoc** [comment]
@MarkusOla  Just to double check, are you doing this assignment alone? I can only see your commits and no approval from your partner.

File: src/pa02/chutes_simulation.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+    def single_game(self):
+        """
+        simulates a single game
+
+        :return: Number of throws for a single player.
+        :return: which type for a given player
+
+        """
+        moves = 0
```
Comment (huynhngoc) [inline src/pa02/chutes_simulation.py]: Would be better if moves is an attribute of Player. Anyway, this is okay.

**Comment by MarkusOla** [comment]
No, it is no problem! 
I know we are supposed to work on different computers, and switch back and forth. But we are working together on my computer. No worries!

**Comment by huynhngoc** [comment]
@MarkusOla  @Rasmussvebestad  We expect you two to switch role regularly (ie, both should have similar number of commits and contributions to prove that both work on the project). And this is really important for the Jan/Jun block project. I need to talk with other TAs about this problem and come back to you later about your grades.

**Comment by MarkusOla** [comment]
Thank you! We will remember to commit from both laptops in Januar!

**Pull request merged**